CMakeLists.txt

```
@@ -63,9 +63,13 @@ if("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU")
   set(PROJ_C_WARN_FLAGS ${PROJ_common_WARN_FLAGS}
     -Wmissing-prototypes
   )
+  if (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 8)
+	  set(PROJ_CXX_WARN_FLAGS ${PROJ_common_WARN_FLAGS}
+		-Wextra-semi
+	  )
```

Removing tabulation characters:
```suggestion
        set(PROJ_CXX_WARN_FLAGS ${PROJ_common_WARN_FLAGS} -Wextra-semi)
```